demisto_sdk/commands/common/hook_validations/image.py

```diff
@@ -199,8 +199,13 @@ def is_not_default_image(self):
     def is_valid_image_name(self):
```

Comment:
@Shellyber How are you certain this only runs on integration and images and not also on other images?

Reply:
Other images? like what? 
This runs as part of the validate_manager which runs on our files (packs).

Reply:
I mean that you also have other types of images (not only integrations - for docs/author etc...), other types of images don't require the naming constrains we are adding now

Reply:
It won't run on those. 
The author image has it's own validation and the others we don't validate at all. its just for our packages - won't affect others